api/rpc/account/account.go

```diff
@@ -215,6 +216,7 @@ func (s *Server) GetUser(ctx context.Context, in *GetUserRequest) (*GetUserReply
 }
 
 // ListUsers implements account.ListUsers
+// nolint : dupl
```

Comment:
Yes, but I would argue that's the reason for tuning. If you know the nature of the code base is going to trigger a lot of false positives, you tweak the threshold a bit. I do think dupl is overly aggressive.